docs/tutorials/deploy_openfaas_workload.md

```diff
@@ -0,0 +1,242 @@
+# OpenFaaS extended workload
+
+## Install
+
+In this tutorial, we assume you have already successfully installed rudr. If not, you could see the [Installation Guide](../setup/install.md) to get help.
+
+After rudr installed, we still need to install [OpenFaaS Operator](https://github.com/openfaas-incubator/openfaas-operator), that will be the real workload executor.
+
+The installation steps for OpenFaaS Operator are almost the same with the guides in the [OpenFaaS Operator README docs](https://github.com/openfaas-incubator/openfaas-operator#deploy-openfaas-with-the-operator).
+
+The only difference you should care about is that the functionNames should use the same namespace you have installed rudr. The namespace is `default` in rudr's helm chart, if you haven't manually changed it, we should use `default` here.    
```

Comment:
These three paragraphs above are verbose as you have listed detailed installation steps below. 

I'd suggest skip them, and add comments in the installation steps.

Reply:
ok